### PR TITLE
Different payment methods support

### PR DESCRIPTION
**Pull request by victorhmp**
#### What is the purpose of this pull request?
Add better support for payments using multiple credit cards, cash and bank invoice.
Update design for `PaymentMethod` component

#### What problem is this solving?
`PaymentSummary` was limited to credit card payments.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Payment/PaymentMethod.js [hunk 1]
```diff
@@ -17,28 +22,24 @@ const PaymentMethod = ({ payment, intl }) => {
...
-            id={'payments.creditcard.lastDigits'}
-            values={
-              { lastDigits: payment.lastDigits }
-            }
-          />
+    <div className="flex justify-between items-center">
+      <div>
+        <p className="t-body lh-copy c-muted-1">
+          <span className="c-on-base">{paymentGroupSwitch(payment.group, intl)}</span>
+          <br />
```
Comment (nandoacoelho) [inline react/components/Payment/PaymentMethod.js]: se vc precisa de um `<br/>`, pq não fazer um outro `<p>`?

File: react/components/Header/Header.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ const Header = ({ data, profile, intl }) => {
         { intl.formatMessage({ id: 'header.thanks' }) }
       </p>
 
-      <p className="center mt4 t-body tc c-muted-1">
+      <p className="center mt4 t-body tc c-muted-1 lh-copy">
         <FormattedMessage
           id={'header.email'}
```
Comment (salesfelipe) [inline react/components/Header/Header.js]:
```suggestion
          id="header.email"
```

File: react/components/Payment/PaymentSummary.js [hunk 3]
```diff
@@ -1,24 +1,14 @@
...
-              <IconPlusLines />
-            </div>
-            <PaymentMethod payment={payment} />
-          </div>
-        ) : (<PaymentMethod payment={payment} key={index} />)
-      })
-    }
-  </div>
+  paymentsData.map((payment, index) => (
+    <div key={index} className="flex flex-column pb6">
```
Comment (klzns) [inline react/components/Payment/PaymentSummary.js]: Precisa desse componente? Nao pode ta inline no componente que usa o `PaymentSummary`?

File: react/components/Payment/PaymentMethod.js (hunk 1 above)
Comment (victorhmp) [reply react/components/Payment/PaymentMethod.js]: Verdade! hahaha
Atualizado :)

File: react/components/Payment/PaymentSummary.js (hunk 3 above)
Comment (victorhmp) [reply react/components/Payment/PaymentSummary.js]: Pode sim! Só criei um componente para ficar mais modular.
Vou colocar no `OrderInfo`, que é quem usa o `PaymentSummary`.

Comment (victorhmp) [reply react/components/Payment/PaymentSummary.js]: Atualizado :)

**Pull request merged**